docs/lib/predictions/fragments/android/getting-started.md

```diff
@@ -0,0 +1,171 @@
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    },
+                    "speechGenerator": {
+                        "voice": "Salli",
+                        "language": "en-US",
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                },
+                "interpret": {
+                    "interpretText": {
+                        "region": "us-west-2",
+                        "defaultNetworkPolicy": "auto"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+Add both the `amplifyconfiguration.json` and the `awsconfiguration.json` to your project inside the `res/raw` directory.
+
+## Configure your application
+
+Add the following dependencies to your **app**'s' `build.gradle` file and click "Sync Now" when prompted:
+
+```gradle
+dependencies {
+    implementation 'com.amplifyframework:core:0.10.0'
+    implementation 'com.amplifyframework:aws-predictions:0.10.0'
+    implementation 'com.amazonaws:aws-android-sdk-mobile-client:2.16.+'
+}
+```
+
+Add the following compile options in the same file to support the Java 8 features that Amplify uses:
+
+```gradle
+android {
+  compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
+```
+
+## Initialize Amplify
+
+Add the following code to your `MainActivity#onCreate` method (ideally this would go in your Application class but this works for getting started quickly):
+
+```java
+AWSMobileClient.getInstance().initialize(getApplicationContext(), new Callback<UserStateDetails>() {
+    @Override
+    public void onResult(UserStateDetails userStateDetails) {
+        try {
+            Amplify.addPlugin(new AWSPredictionsPlugin());
+            Amplify.configure(getApplicationContext());
+            Log.i("PredictionsQuickstart", "All set and ready to go!");
+        } catch (Exception exception) {
+            Log.e("PredictionsQuickstart", exception.getMessage(), exception);
+        }
+    }
+
+    @Override
+    public void onError(Exception exception) {
+        Log.e("PredictionsQuickstart", "Initialization error.", exception);
+    }
+});
+```
+
+## Working with the API
+
+Here is an example of translating text. In order to override any choices you made in regards to target or source languages while adding this resource using the Amplify CLI, you can pass in them in directly as parameters as shown below.
+
+```java
+Amplify.Predictions.translateText(
+        "I like to eat spaghetti",
+        LanguageType.ENGLISH,
+        LanguageType.SPANISH,
+        result -> Log.i("PredictionsQuickstart", result.getTranslatedText()),
+        error -> Log.e("PredictionsQuickstart", error.toString())
+);
+```
+
+As a result of executing this code, this you should see the following printed to your console:
```

Comment:
Should or will?

Reply:
"will" sounds better haha. "This code is tested and approved"